### ISSUE TITLE
[ICC] [Item] Unidentifiable Organ

### ISSUE THREAD

**Issue by ripsnarl** [dalaranwow/dalaran-wow#4740]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)
The trinket only procs on blocks rather than just melee attacks.

**Expected behavior**: Tell us what should happen instead
as said in the description of the trinket, each time you are struck by a melee attack, you have a 60% chance to gain 24 stamina for the next 10 seconds, stacking up to 10 times

**Steps to reproduce the problem**:

1. equip unidentifiable organ
2.  receive melee attacks

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
https://www.youtube.com/watch?v=er2uUCcbP38 
as you can see it is proc'ing for a druid tank, who doesn't have the ability to block

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
item id: 50341

**Include Screenshots from the issue if necessary**

**Comment by SadhuArch** [comment]
_Just thought I'd add a very exciting movie of me fighting a Rhino in Storm Peaks to add to the cause._ As you can see the Invigorated buff proc only occur on blocks. It's very easy to see if you keep an eye on the combat text. It doesn't happen on every block but that's normal I guess, seeing how the proc rate is 60%.

https://www.youtube.com/watch?v=YlBWLt9NyQ8

**Comment by SadhuArch** [comment]
No attention on this issue? No labels or anything yet. What's happening? Feels like it's an easy fix.

**Comment by Rushor** [comment]
DB/Item: Unidentifiable Organ - Now proccs on blocks and on melee hit taken

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4740
* commit: e815e62ae4113a7d3006da16d5230b4bf41c8fac

The issue was fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
